### ISSUE TITLE
Getting "nan" values for loss function

### ISSUE THREAD

**Issue by kvnptl** [Singingkettle/SAF-FCOS#3]
Hello @Singingkettle,

I am attempting to reproduce the results on my end by training a model as per the instructions in the README file. However, for the full 40000 iterations of training on a single GPU, I encountered "nan" values for the loss.

As mentioned in the paper,

> In addition, the output values of loss function are always “nan” in early iterations. Worse still, we cannot train the SAF-FCOS model easily even if we restart the training code multiple times. However, when we set the number of residual blocks in the first stage as 1, we may need to make a few tries to restart the training code so as to help the detection model to get rid of “nan” in losses. 

I tried a few times with restart but still getting "nan" values, Could you please share some insights on this? on how to get proper values?

Thanks,
K

**Comment by kvnptl** [comment]
Now, I am getting loss values instead of "nan". I think the problem was it was taking previous weights where it ended up with "nan" values. And once it is entered in "nan" values, it can never recover. 

So, I deleted previous run weights (.pth), and it started working fine.
